sv_interval.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, chrom=None, start=0, end=0, name=None, sv_type=None, length=0
     self.is_precise = False
     self.is_validated = False
     self.validating_interval = None
-    self.info_dict = {}
+    self.info_dict = {} # Marghoob, did you have any specific plans for this?
```

Comment:
It was supposed to mirror info in the input VCFs. I was planning on retaining the original information. Now that we are keeping the original stuff in native_sv field, this doesn't seem necessary.